### ISSUE TITLE
Factors affecting MPG 

### ISSUE THREAD

**Issue by HisenZhang** [HisenZhang/IED-tranport-sim#1]
Figure out how the following factors affect fuel efficiency (Miles per Gallon):
- inclined road
- cargo (and engine/power source) load
- current velocity

**Comment by Liamtracey** [comment]
inclined road: for every 1% incline, car comsumes roughly an extra .4 gallons per 100 miles
[https://afdc.energy.gov/data/10601#:~:text=On%20an%20uphill%20grade%2C%20all,%2D3%25%20to%205%25.&text=Both%20conventional%20and%20hybrid%20SUVs,likely%20because%20they%20are%20heavier.](url)

**Comment by Liamtracey** [comment]
Cargo load: every 10,000 pounds of cargo makes you lose half a mile per gallon in fuel efficiency. 
[https://www.imiproducts.com/blog/calculate-fuel-efficiency-gains/#:~:text=Going%20back%20to%20the%20Starship,half%20a%20mile%20per%20gallon.&text=The%20more%20cargo%20you%20haul,resulting%20in%20more%20fuel%20saved.](url)

**Comment by Liamtracey** [comment]
same applies to engine/power source

**Comment by Liamtracey** [comment]
Velocity: ideal velocity for efficiency - 55 mph
Fuel efficiency decreases after 55 mph 
[https://www.mpgforspeed.com/#:~:text=Unfortunately%2C%20it's%20true.,is%20around%2055%2D60%20mph.&text=You'd%20be%20surprised%20to,minutes%20to%20your%20travel%20time.](url)

**Comment by connorsammut** [comment]
According to FuelEconomy.gov, the 2018 Tesla Model S Standard Range requires about 31 kWh per every 100 miles
https://news.energysage.com/what-does-it-cost-to-charge-a-tesla-with-solar-energy/

**Comment by connorsammut** [comment]
90kWh battery weighs 60lbs
https://stealthev.com/product/tesla-6-2-kwh-module-from-90kwh-pack/

**Comment by connorsammut** [comment]
Tesla Solar Panel Specs:
315 W
Dimensions
66.3" x 39.4" x 1.57"
Weighs 44.5 lbs
I did the math we can fit around 8 of these on the roof of the cargo van

**Comment by connorsammut** [comment]
https://www.tesla.com/solarpanels

**Comment by HisenZhang** [comment]
> inclined road: for every 1% incline, car comsumes roughly an extra .4 gallons per 100 miles
> https://afdc.energy.gov/data/10601#:~:text=On%20an%20uphill%20grade%2C%20all,%2D3%25%20to%205%25.&text=Both%20conventional%20and%20hybrid%20SUVs,likely%20because%20they%20are%20heavier.

This does not seem to be a good estimation.

**Comment by connorsammut** [comment]
Yeah you’re right that graph makes it look like for 3% uphill the truck van
would consume around 2.5 extra gallons per 100 miles and for 3% down hill
the van would consume around 1 gallon less per 100 miles.
